### DISCUSSION TITLE
Add Contributor's details in Crowdfund (or at least email), same like in Point of Sale

### DISCUSSION THREAD

**Discussion by Luca399** [btcpayserver/btcpayserver#4509] [Ideas / Feature Requests]
Hi.
I consider very useful to introduce in Crowdfund, the option to collect Donor's info, or at least the email, the same like in Point of Sale. This way, we can know who contributed and also give "special thanks" to the donors who contributed more, or deliver the services we can promise in Perks. Also would be useful to be able to see each donor for what perks they donated, in this way, as said, in these perks we can include some services/products to offer as "thanks" to each donor, or at least to the biggest ones.

Thanks.

**Comment by bolatovumar** [comment]
@Luca399 if you select `Requires a refund email` in store settings under the "Checkout Appearance" tab you will then be able to collect the email address on checkout.
